package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "devDependencies": {
     "@types/jest": "24.0.3",
+    "@types/jest-diff": "^20.0.1",
```

Comment:
do we have a build and test dependency CQ ?

Reply:
removed, thank you!